LLMonFHIR/FHIR Display/FHIRResourcesView.swift

```diff
@@ -46,14 +70,49 @@ struct FHIRResourcesView: View {
                     }
                 }
                 .toolbar {
+                    ToolbarItem(placement: .navigationBarTrailing) {
+                        Button(action: {
+                            if !interpretingMultipleResources {
+                                showMultipleResourcesChat.toggle()
+                            } else {
+                            }
+                        }) {
+                            if interpretingMultipleResources {
+                                ProgressView("Interpreting...")
+                            } else {
+                                Text("Chat with All Records")
+                            }
+                        }
+                    }
```

Comment:
While I like to first focus on the functionality of the app, I would already have one first suggestion on how to improve the UI a bit to feel a bit more like the current UI design of the app and how Apple designs their apps. While the toolbar placement works for one item, multiple trailing toolbar items are not often used in mobile applications. 

I would therefore suggest placing the button a bit more prominent and centered in the UI, maybe something like this?
<img width="370" alt="Screenshot 2023-07-11 at 8 42 29 PM" src="https://github.com/StanfordBDHG/LLMonFHIR/assets/28656495/4a406e70-8d35-4e54-868e-1d57e2f8b66a">

This could, e.g. be achieved using the `OnboardingActionsView` from the SpeziOnboarding package and placed above the list of resource types (currently line [53](https://github.com/StanfordBDHG/LLMonFHIR/pull/23/files#diff-8c1ada103658e4da8f3b8d89b13392bd55097d5ef5e0e5aa6dea39dbd4a8b733R53)) 

```swift
Section {
    OnboardingActionsView(
        "CHAT_WITH_ALL_RESOURCES",
        action: {
            await interpretMultipleResources()
            showMultipleResourcesChat.toggle()
        }
    )
        .padding(-20)
}
```

I would also suggest a slightly different flow of events in the app.
Currently the `interpretMultipleResources()` function is called right after the view is loaded. I would suggest that we only call this function when the user wants to chat about all resources and e.g. presses the button like shown in the button implementation above. 
The best solution would be if we already show the chat view right after the user pressed the button, then load the query in the background, and then populate the chat. That might not be needed in this version of the PR and might play well with the following feature suggestion: https://github.com/StanfordSpezi/SpeziML/issues/15

Reply:
Sounds great! I removed the button from the tool bar and added the ```OnboardingActionsView``` button to the top of the list. I used the same action which you provided so that ```interpretMultipleResources()``` is only called when the button is tapped.